lib/Sema/TypeCheckDecl.cpp

```diff
@@ -4127,8 +4127,19 @@ class DeclChecker : public DeclVisitor<DeclChecker> {
             TC.checkTypeModifyingDeclAttributes(var);
 
           // Decide whether we should suppress default initialization.
-          if (!PBD->isDefaultInitializable(i))
-            continue;
+          //
+          // Note: in Swift 4, we screwed up: a property of Optional<Int> does
```

Comment:
Please don't add comments like this that tell a personal story of the team.  Instead keep it factual:

// Note: The Swift 4 compiler contained a bug that ...